### DISCUSSION TITLE
Can't connect to Intellij idea

### DISCUSSION THREAD

**Discussion by zimabueter** [community/community#7264] [Copilot Conversations] [answered]
First when I tried I accidently closed browser windows , after that IDE do not suggest to connect tried reinstall plugin didn't work

**Comment by jansorg** [comment, marked as answer]
Which IDE do you use?

**Reply by zimabueter** [reply]
IntelliJ IDEA

**Reply by jansorg** [reply]
You can re-login by opening `Tools > GitHub Copilot > Login to GitHub`.

**Reply by zimabueter** [reply]
Thanks <3
